pipelines/DRP.yaml

```diff
@@ -220,6 +220,28 @@ subsets:
       The multiVisit subset defined in pipe_tasks' DRP.yaml is not safe to
       use on HSC for various reasons; use 'step1', 'step2', and 'step3' subsets
       instead.  It may be re-enabled in the future.
+  singleFrame:
+    subset:
+      - isr
+      - characterizeImage
+      - calibrate
+      - writePreSourceTable
+      - transformPreSourceTable
+      - consolidatePreSourceTable
+      - consolidateVisitSummary
+      - skyCorr
+      - isolatedStarAssociation
+    description: >
+      Set of tasks for complete single frame processing. Analogous to
+      SingleFrameDriver.
```

Comment:
I'm not sure we should mention SingleFrameDriver here: it's gen2-only, and most people won't know what it's referring to. Maybe a statement that because this includes skyCorr, it needs some large-ish number of detectors in each visit?